test/templates/api/repositories/template-preview-unknown-variable/002-poll/expected-response.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "template.preview/%s",
+  "type": "template.preview",
+  "url": "https://templates.keboola.local/v1/tasks/template.preview/%s",
+  "status": "error",
+  "isFinished": true,
+  "createdAt": "%s",
+  "finishedAt": "%s",
+  "duration": %d,
+  "error": "%s Unknown variable: InstanceIdShort"
```

Comment:
Is the `InstanceIdShort` variable of function? Can you check jsonnet whether the `functions` and `variables` are different things? In case it is the same just resolve the comment.